exercises/simple-linked-list/.meta/hints.md

```diff
@@ -0,0 +1,35 @@
+## Implementation Hints
+
+Do not use the struct `SimpleLinkedList` just as a wrapper around a `Vec`, but use the fields `head` and `len`.  
+This might look similar to this:
+```
+pub struct SimpleLinkedList<T> {
+    head: Option<Box<Node<T>>>,
+    len: usize,
+}
+```
+The `head` field points to the first element (Node) of this linked list.  
+Also implements this struct `Node` with the following fields:
```

Comment:
Instead of "Also implements...", consider phrasing this as "This implementation also requires a struct `Node` with the following fields:"